controllers/tasks.js

```diff
@@ -43,6 +43,33 @@ const fetchTasks = async (req, res) => {
     return res.boom.badImplementation('An internal server error occurred')
   }
 }
+
+/**
+ * Fetches all the tasks of the requested user
+ *
+ * @param req {Object} - Express request object
+ * @param res {Object} - Express response object
+ */
+const getUserTasks = async (req, res) => {
+  try {
+    const { username } = req.params
+
+    if (username) {
+      const allTasks = await tasks.fetchUserTasks(username, [], true)
```

Comment:
We need to handle the case when the `username` provided is wrong. We should be returning 404 if I pass `obama` as username.
Please check this controller function:
https://github.com/Real-Dev-Squad/website-backend/blob/c32f177d61ccce9890b02d662be5c0031c20e71a/models/auctions.js#L146

And this model function:
https://github.com/Real-Dev-Squad/website-backend/blob/c32f177d61ccce9890b02d662be5c0031c20e71a/models/auctions.js#L146

By these two example functions, you'll get how to handle the wrong username case. Otherwise we'd get 5xx error on production which isn't something we want.